src/windows/models/properties_model.py

```diff
@@ -301,7 +301,7 @@ def color_update(self, item, new_color, interpolation=-1, interpolation_details=
                         # Keyframe
                         # Loop through points, find a matching points on this frame
                         found_point = False
-                        for point in clip_data[property_key][color].get('Points', []):
+                        for point in clip_data[property_key][color].get("Points", []):
```

Comment:
:confused: 

Bit of an odd commit — there's nothing _wrong_ with it, per se, but single and double quotes are 100% interchangeable in Python... unlike other languages, there are zero implicit differences between the two. Ditto triple-quoted strings, which can use either `'''` or `"""` as long as they match on both ends.

So, the code wasn't really any less "_safe_" with the single quotes. (Though, as I said, nothing wrong with changing them either, so I'm fine either way.)

Reply:
Or... maybe this was supposed to be `clip_data[property_key].get(color, {}).get("Points", [])`?